### ISSUE TITLE
[v0.11.5] Code generation fails when object has property named 'size'

### ISSUE THREAD

**Issue by joaomlneto** [asyncapi/java-spring-cloud-stream-template#228]
#### Describe the bug
Using this generator version 0.11.5, when attempting to generate code for a message that has a field named `size`, the code generation fails.

Version 0.11.4 works.

The error:
```
Something went wrong:
Template render error: (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/@asyncapi/java-spring-cloud-stream-template/template/src/main/java/$$everySchema$$.java)
  TypeError: Cannot set property size of #<Map> which has only a getter
    at Object._prettifyError (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/lib.js:36:11)
    at /opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:563:19
    at eval (eval at _compile (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:633:18), <anonymous>:46:11)
    at /opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:613:9
    at eval (eval at _compile (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:633:18), <anonymous>:12:11)
    at /opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:613:9
    at Template.root [as rootRenderFunc] (eval at _compile (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:633:18), <anonymous>:367:3)
    at Template.getExported (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:611:10)
    at eval (eval at _compile (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:633:18), <anonymous>:11:5)
    at createTemplate (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:315:9)
    at handle (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:327:11)
    at /opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:339:9
    at next (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/lib.js:328:7)
    at Object.asyncIter (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/lib.js:334:3)
    at Environment.getTemplate (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:321:9)
    at Template.root [as rootRenderFunc] (eval at _compile (/opt/homebrew/lib/node_modules/@asyncapi/generator/node_modules/nunjucks/src/environment.js:633:18), <anonymous>:9:5)
```

#### How to Reproduce
The following schema produces the error:

```yml
asyncapi: 2.2.0
info:
  title: Amazing Service
  version: 1.0.0
channels:
  /amazingness:
    publish:
      message:
        name: My Amazing Message
        payload:
          type: object
          properties:
            size:
              type: integer
```

Generate using:
```ag <SCHEMA> @asyncapi/java-spring-cloud-stream-template -o <DIR>```

#### Expected behavior
Code generation should just work!

#### Workaround
Fall-back to previous version (0.11.4):
```ag <SCHEMA> @asyncapi/java-spring-cloud-stream-template@v0.11.4 -o <DIR>```

**Comment by joaomlneto** [comment]
Issue still unresolved. Still fails with `v0.11.6`, `v0.11.7`, `v0.12.0`, `v0.12.1`, `v0.13.0`.

**Comment by CameronRushton** [comment]
@joaomlneto Thanks for raising this bug and keeping track of major releases. Please confirm that the issue is truly resolved with the referenced fix.

**Comment by joaomlneto** [comment]
Can confirm issue resolved as of `v0.13.2`! 🎉  Thanks! 😄
